<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>&amp;Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Кла&amp;cc</string>
         </property>
         <property name="buddy">
          <cstring>cmbType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="cmbType"/>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>&amp;В Б/Л требуется диагноз</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblFinance">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Группа</string>
         </property>
         <property name="buddy">
          <cstring>cmbGrouping</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QComboBox" name="cmbGrouping">
         <item>
          <property name="text">
           <string>заболевание</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>уход</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>беременность и роды</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLabel" name="lblMaxAge">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Максимальные длительности:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="lblPrimary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>первичного периода ВУТ в днях</string>
         </property>
         <property name="buddy">
          <cstring>spinPrimary</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QSpinBox" name="spinPrimary">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="lblProlongate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>продления ВУТ в днях</string>
         </property>
         <property name="buddy">
          <cstring>spinProlongate</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QSpinBox" name="spinProlongate">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0" colspan="3">
        <widget class="QLabel" name="lblRestriction">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ВУТ, после которого требуется КЭК</string>
         </property>
         <property name="buddy">
          <cstring>spinRestriction</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QSpinBox" name="spinRestriction">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="2" colspan="2">
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>111</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>&amp;Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>cmbGrouping</tabstop>
  <tabstop>spinPrimary</tabstop>
  <tabstop>spinProlongate</tabstop>
  <tabstop>spinRestriction</tabstop>
  <tabstop>tblIdentification</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
